helm/releases-vmware/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
I think you will need more changes to actually make this "deployable"

Reply:
See also here:
https://github.com/giantswarm/releases/blob/760c7d726591e84ded633847c7d6907d0cfbe077/.circleci/config.yml#L29

Reply:
it is highly likely but I cannot really test is anyway :D

Reply:
It should be fixed now :D